pod-e2e.sh

```diff
@@ -10,7 +10,7 @@ POD_NAME="pod_e2e"
 
 echo "cleaning up and recreating pod"
 podman pod rm -f ${POD_NAME}
-podman pod create --name=${POD_NAME} --publish 8000
+podman pod create --name=${POD_NAME} --publish 8000:8000
 
 POD_NAME="pod_e2e" ./backend/tests/opensearch_ocp.sh
```

Comment:
If you `export POD_NAME="pod_e2e"` then you don't have to redefine the environment variable here to pass it to the script.

Also, I *generated* a random pod name, so that multiple simultaneous pods won't conflict: `export POD_NAME=${POD_NAME:-FUNC${RANDOM}}`. (If you'd prefer `e2e` instead of `FUNC`, that's fine, although "functional test" is a more flexible and general term.)

Reply:
i like the idea of appending a random string

Reply:
This is also being used as a developer utility to facilitate development of GUI functional tests. To make it support both of the needs of CI and local development, then it would need a cleanup option and logic to remove the pod name with a random string. It's just not necessary at the moment.